include/sys_math.h

```diff
@@ -0,0 +1,13 @@
+#ifndef SYS_MATH_H
+#define SYS_MATH_H
+#include "ultra64.h"
+
+f32 Math_FactorialF(f32 n);
+f32 Math_Factorial(s32 n);
+f32 Math_PowF(f32 base, s32 exp);
+f32 Math_SinF(f32 angle);
+f32 Math_CosF(f32 angle);
+
+#include "sys_math_atan.h"
```

Comment:
There doesn't necessarily have to be 1 header for 1 source file if the source files essentially implement the same system. I think you should get rid of `sys_math_atan.h` and move its contents here.